samples/bluetooth/llpm/README.rst

```diff
@@ -0,0 +1,313 @@
+   It demonstrates the performance of an LLPM-enabled connection that the receiver will receive the data approximately every 1 ms.
+
+GATT Latency Service
+   To measure the transmission latency from application layers, a GATT Latency service `BT_UUID_LATENCY` is included to compute the time spent.
+   When the sender writes its timestamp to the `BT_UUID_LATENC_CHAR` characteristic of the receiver, the Latency service of the receiver will automatically reply back.
+   Whenever the sender receives a response, it will use its present time and the corresponding timestamp written before to estimate the round-trip time (RTT) of a writing characteristic procedure (see `Bluetooth Core Specification`_: Vol 3, Part F, 3.4.5 Writing attributes).
+
+.. list-table:: GATT Attributes
+   :header-rows: 1
+
+   * - Type
+     - UUID
+     - Property
+   * - Primary service
+     - BT_UUID_LATENCY
+     - Read only
+   * - Characteristic
+     - BT_UUID_LATENCY_CHAR
+     - BT_GATT_CHRC_WRITE, BT_GATT_PERM_WRITE
+
+This sample transmits data between two boards to measure the transmission latency in between.
+One of the devices is connected as a *master* and another is connected as a *slave*.
+The performance is evaluated with the transmission latency dividing the estimated round-trip time in half (RTT / 2).
+
+By default, the following values are used to demonstrates the interaction of the connection parameters:
+
+.. list-table:: Default parameter values
+   :header-rows: 1
+
+   * - Parameter
+     - Value
+   * - Connection interval
+     - 80 units (100 ms)
+   * - LLPM connection interval
+     - Lowest interval (1 ms)
+   * - Physical layer (PHY)
+     - LE 2M PHY
+
+
+Requirements
+************
+
+* Two of the following nRF52-series development kit boards:
+
+  * PCA10040
+  * PCA10056
+
+  You can mix different boards.
+* Connection to a computer with a serial terminal for each of the boards.
+
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/llpm`
+
+.. include:: /includes/build_and_run.txt
+
+
+Testing
+=======
+
+After programming the sample to both boards, test it by performing the following steps:
+
+1. Connect to both boards with a terminal emulator (for example, PuTTY).
+   See :ref:`putty` for the required settings.
+#. Reset both boards.
+#. Observe that the boards establish a connection.
+   When they are connected, one of them serves as *master* and the other one as *slave*.
+
+   - The master outputs the following information::
+
+       Press any key to set LLPM short connection connection interval (1 ms)
+
+   - The slave outputs the following information::
+
+       Press any key to start measuring transmission latency
+
+#. Press a key in the terminal that is connected to the slave.
+#. Observe that the slave starts sending data to the master and measuring the transmission latency with the default connection interval::
```

Comment:
Would rephrase to:
```suggestion
#. Observe that latency measurements are printed in the terminal. The latency is expected to be slightly shorter than the connection interval:
```

Reply:
As per RST formatting, the colon needs to be double `::`. Otherwise, the RST will treat the single colon as a linking markup and break the formatting.